src/raft_server.c

```diff
@@ -6353,7 +6371,9 @@ raft_server_instance_run(const char *raft_uuid_str,
 int
 raft_server_get_leader_ts(struct raft_leader_ts *leader_ts)
 {
-    struct raft_instance *ri = raft_net_get_instance();
+    struct raft_instance *ri = raft_server_global_instance.ri;
```

Comment:
Might as well use a method here for getting the raft instance instead of accessing it directly.